frontend/js/sntmGeo.js

```diff
@@ -0,0 +1,55 @@
+// async function getAllPoems () {
+//   const res = await fetch('/api/poems')
+//   const data = await res.json()
+//   data.forEach(poem => {
+//     poem.author
+//   });
+// }
+
+// Sentiment Analysis package (installed from npm)
+async function getSentiment (text) {
+  const Sentiment = require('sentiment')
```

Comment:
sentiment is a node.js module meant to be used on the backend (this won't work on the frontend), the `require()` function is how we import modules (ie. libraries) on the backend, but that function doesn't exist on the frontend (ie. on the "client side", meaning it won't work in a browser, only on a server)

what u would want to do here is create an endpoint on our REST API which u can query for sentiment data by `fetch()`ing for it on the frontend

Reply:
Got it, I'll work on that now.